src/python_testing/TC_SC_4_3.py

```diff
@@ -152,6 +152,14 @@ def verify_t_value(self, t_value):
                 return True, f"T value ({t_value}) is valid and bit 0 is clear."
             else:
                 return False, f"Bit 0 is not clear. T value ({t_value})"
+
+            # Check that the value can be either 2, 4 or 6 depending on whether
+            # DUT is a TCPClient, TCPServer or both.
+            if T_int & 2 == 1 or T_int & 4 == 1:
```

Comment:
is this ever true? The & as bitflags seems odd to compare with 1. Is binary and not `foo & 2` can only be 2 ? and same for `4 can only be 4` and never one?

Could we use bools (like `!= 0` ?) to make it easier to read and validate?

Reply:
Oh! Yes. Thanks for catching that! Fixed. Not sure how it was passing though.